includes/validation/class-amp-validation-manager.php

```diff
@@ -1963,7 +1944,7 @@ public static function get_validate_url_error_message( $error_code, $error_messa
 
 		$site_health_message = '';
 		if ( version_compare( get_bloginfo( 'version' ), '5.2', '>=' ) ) {
-			$site_health_message .= ' ' . sprintf(
+			$site_health_message .= sprintf(
 				/* translators: %s is link to Site Health */
 				__( 'Please check your <a href="%s">Site Health</a> to verify it can perform loopback requests.', 'amp' ),
```

Comment:
Translations should be escaped, as the actual HTML that will be sent to the browser is crowd-sourced user-input.

Reply:
See d48ceae6f50623a6e0abb63057f7dd294e98d54c